### PR TITLE
Fixed a handful of typos

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

File: Doc/library/ssl.rst [hunk 1]
```diff
@@ -947,8 +947,7 @@ SSL Sockets
    the specification of normal, OS-level sockets.  See especially the
    :ref:`notes on non-blocking sockets <ssl-nonblocking>`.
 
-   Usually, :class:`SSLSocket` are not created directly, but using the
-   the :meth:`SSLContext.wrap_socket` method.
+   Usually, :class:`SSLSocket` are not created directly, but using the :meth:`SSLContext.wrap_socket` method.
```
Comment (berkerpeksag) [inline Doc/library/ssl.rst]: This line is a bit long. It would be nice to wrap it at 80 or so chars.

Comment (alex) [reply Doc/library/ssl.rst]: fixed

**Review by marco-buttu — APPROVED** [review]
Thank you, LGTM :-)

**Review by berkerpeksag — APPROVED** [review]

**Comment by Mariatta** [comment]
So I'm not sure how to apply the labels here lol 

Doc/library/email.compat32-message.rst -> needs backport to 3.6
Doc/library/email.message.rst -> needs backport to 3.6
Doc/library/idle.rst -> needs backport to 2.7, 3.5, 3.6
Doc/library/ssl.rst -> needs backport to 3.6

**Comment by berkerpeksag** [comment]
I'd say don't bother to backport this to 2.7 and 3.5 :)

**Comment by Mariatta** [comment]
ok. then everything else LGTM :)

**Review by Mariatta — APPROVED** [review]

**Pull request merged**